### ISSUE TITLE
Stopped Working

### ISSUE THREAD

**Issue by christopher-paul-shaw** [shivammathur/setup-php#205]
Been using for php 5.6 but stopped working today

        Run shivammathur/setup-php@v1
        /bin/sh /opt/hostedtoolcache/linux.sh 5.6 /home/runner/work/_actions/shivammathur/setup-php/v1/dist

        ==> Setup PHP
        /opt/hostedtoolcache/linux.sh: 225: /opt/hostedtoolcache/linux.sh: php5.6: not found
        ✓ PHP Installed PHP 

        ==> Setup Tools
        ✓ composer Added

when moving onto the builkd step after the above, its trying to run as 7.4

**Comment by shivammathur** [comment]
The runner has broken packages, please add a step to update the packages before the setup PHP step.
```yml
- name: Update
  run: sudo DEBIAN_FRONTEND=noninteractive apt-get update
```

**Comment by shivammathur** [comment]
Created a issue to get this fixed actions/virtual-environments#592

**Comment by shivammathur** [comment]
@christopher-paul-shaw Patched the release, It should work now without errors, same as before.
Will revert the patch once fixed on the runner.
